src/Bicep.LangServer/Handlers/BicepHoverHandler.cs

```diff
@@ -68,6 +68,9 @@ public override Task<Hover> Handle(HoverParams request, CancellationToken cancel
                 case ResourceSymbol resource:
                     return $"```bicep\nresource {resource.Name}\n{resource.Type}\n```";
 
+                case ModuleSymbol module:
+                    return $"```bicep\nmodule {module.Name}\n{module.Type}\n```";
```

Comment:
Will `module.Type` be set to "module" or the relative path?

Reply:
Discussed offline - I'm going to set it to the path